config/overlays/istio/registry-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
i opened a draft for this namespace https://github.com/opendatahub-io/opendatahub-operator/pull/930
see description for all the labels created there. 
i do  not think you will need `sidecar.istio.io/inject: "true"` but `istio-injection=enabled` if that is what you want

Reply:
Good catch, it's the istio-injection label.